extensions/adobe/experience/primetime-authentication/application.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":   "https://ns.adobe.com/experience/primetime-authentication/application",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Application extensions for Adobe Primetime Authentication",
+  "type": "object",
+  "meta:extends": ["https://ns.adobe.com/xdm/context/application"],
+  "description":
+    "Application extension properties specific to Adobe Primetime Authentication.",
+  "definitions": {
+    "application": {
+      "properties": {
+        "xdm:clientAPIVersion": {
+          "title": "Client API Version",
+          "type": "string",
+          "description":
+            "The client API version embedded or used by the application."
+        },
+        "xdm:serviceAPIVersion": {
+          "title": "Server API Version",
+          "type": "string",
+          "description":
+          "The server API version to which the application connects to."
+        },
+        "xdm:serviceProvider": {
+          "title": "Service Provider",
+          "type": "string",
+          "description":
+            "The server API version to which the application connects to."
+        },
+        "xdm:deviceType": {
```

Comment:
With these two "deviceTypes" can we look at improving the standard XDM device type to cover the needs?

Reply:
The owner of the application is using this field to identify/track the device/platform in their metrics. The value is free form and ranges from user friendly names ("android-tv") to random generated values ("113e0b30330c5415e033" which is "fire-tv")